src/components/common/styled-components.js

```diff
@@ -540,3 +553,33 @@ export const MapControlButton = styled.div.attrs({
     color: ${props => props.theme.secondaryBtnActColor};
   }
 `;
+
+export const StyledFilterPanel = styled.div`
```

Comment:
`StyledFilterPanel` is also defined in `filters/components` and `side-panel/filter-panel/filter-panel`.  Can you delete duplicates and keep them all in one place? I don't think this should be a generic `styled-components` I would keep it under `side-panel`

Reply:
i removed the code from here. I think i have started creating some generic components and stopped at some point